FixMatrix/FixMatrix.cpp

```diff
@@ -0,0 +1,208 @@
+#include <cstdlib>
+#include <iostream>
+#include <stdexcept>
+
+namespace {
+
+    class matrix {
+        int Row, Col;
+        double **Value;
+    public:
+
+        matrix(unsigned int, unsigned int);
+
+        matrix(const matrix &);
+
+        int GetRow();
+
+        int GetCol();
+
+
+        friend bool operator==(const matrix &m1, const matrix &m2);
```

Comment:
Well, i changed my mind. Lets make these overloaded operators as regular functions below the class definition, not the friend ones